js/samples/quickstart-csharp-azfunction/Readme.md

```diff
@@ -0,0 +1,53 @@
+# Immersive Reader - Quickstart C# calling Azure Function
+
+This code sample shows how to get Azure AD token from an Azure function.
+
+## Prerequisites
+
+* A function already created and deployed into Azure, see this [Azure function code sample](https://github.com/microsoft/immersive-reader-sdk/tree/master/js/samples/azure-function-csharp).
+        
+    You must have the Azure function code sample setup and running to get the `YOUR_FUNCTION_LOCAL` value to be used in the next section.
+
+    Follow the steps in the sample and run the function either pressing **F5** or with the command below:
+
+    > func host start
+
+    You must deploy the function to Azure first to get the `YOUR_FUNCTION_URL` value.
```

Comment:
steps for local are mentioned and just added some reference in case they want to deploy it to azure.
Link to github sample source code is added as well for reference.